devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy

```diff
@@ -271,6 +278,21 @@ private Map getMoistureResult(value) {
 	]
 }
 
+/**
+ * PING is used by Device-Watch in attempt to reach the Device
+ * */
+def ping() {
+
+	if (state.lastActivity < (now() - (1000 * device.currentValue("checkInterval"))) ){
+		log.info "ping, alive=no, lastActivity=${state.lastActivity}"
+		state.lastActivity = null
+		return zigbee.readAttribute(0x001, 0x0020) // Read the Battery Level
```

Comment:
After inferring that the device isn't alive, why are we returning zigbee.readAttribute(0x001, 0x0020) i.e. Reading the battery level.
Is it because calling the 'refresh()' is a overhead when we are just trying to update the lastActivity time?